plugins/ida_binsync/ida_binsync/controller.py

```diff
@@ -193,6 +193,9 @@ def binary_path(self) -> Optional[str]:
     def get_func_size(self, func_addr) -> int:
         return compat.get_func_size(func_addr)
 
+    def goto_address(self, func_addr) -> None:
```

Comment:
@zachsez could you comment this? So people looking at IDA API in the future know wtf we were going here. Just briefly describe what this code does to IDA.